### ISSUE TITLE
Internal Server Error

### ISSUE THREAD

**Issue by DMF194** [microsoft/sample-app-aoai-chatGPT#840]
**Describe the bug**
After I git clone into my own private branch and running all the instructions from the READ.ME, I got the internal server error

**To Reproduce**
I am following the instructions for local machine deployment
1. Fill the details for my .env file as follows
 - AZURE_OPENAI_RESOURCE
 - AZURE_OPENAI_MODEL
 - AZURE_OPENAI_KEY
 
2. Fill the details for my .env file as follows
- AZURE_SEARCH_SERVICE
- AZURE_SEARCH_INDEX
- AZURE_SEARCH_KEY

3. Made sure that my startup command is as follows:
python -m gunicorn app:app under web service configuration tab

**Screenshots**
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/82444697/e6b718b7-5040-4b7f-91d8-7608843ffc9d)

Note that I am doing this using company proxy and not using chat history.

**Comment by DMF194** [comment]
Just an update to this case, I tried to enter the cosmos details and restarted the application.

The following screenshot is shown
![image](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/82444697/d8fc0b4e-cfec-4c60-91a1-89c98bbe9fb1)

**Comment by DMF194** [comment]
Hi @abhahn, when I tried restarting the application by running the start.cmd command this morning.

I no longer run into the internal server issue. 

This is what it looks like today

![IMG_20240524_091001](https://github.com/microsoft/sample-app-aoai-chatGPT/assets/82444697/3202dc92-1118-4267-a061-34d4115fa5bc)


But comparing both the debug log from previous runs and today, I noticed the httpx python package is not throwing errors.

Previously, when I tested the conversation API by clicking on the send button, 500 error code kept appearing. My assumption is that the httpx python package used had some issues.
